academic_validation_framework/reporting/base.py

```
@@ -0,0 +1,260 @@
+"""
+Base report generator interface.
+"""
+
+import json
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)